.github/workflows/main.yml

```diff
@@ -24,9 +24,12 @@ jobs:
         uses: jurplel/install-qt-action@v2
       - name: Build Qt
         run: |
-          lrelease src/FrogArchiver.pro
-          qmake -project && qmake src/FrogArchiver.pro CONFIG+=release && make
-        
+          cmake -S. -Bbuild
+          cmake --build build --target package
+      - uses: actions/upload-artifact@v3
+        with:
+          name: my-artifact
+          path: ./*-Linux.tar.gz
```

Comment:
use build/*.tar.gz

Reply:
All the generated stuff is in there due to the -B switch we have used in the cmake configure step